docs/api/RegistrationClose2.md

```diff
@@ -0,0 +1,46 @@
+RegistrationClose2 function
+======
+
+> **Preview**
+> This routine is in preview and is subject to breaking changes.
+
+Closes an existing registration asynchronously.
```

Comment:
Instead of calling it RegistrationClose2, why not RegistrationCloseAsync, to indicate that it's specifically for async usage?

Reply:
RegistrationClose does substantially different work from RegistrationShutdown, and while I'm not sure the work can't be migrated from Close to Shutdown, I also don't want to change that behavior for a preview feature. For now, I think the RegistrationClose2 unblocks our AFP partner without risking breaking changes to the rest of the ecosystem; we should be able to carefully refactor that preview API in the future.